src/vstart.sh

```diff
@@ -1785,11 +1799,13 @@ do_rgw()
     for n in $(seq 1 $CEPH_NUM_RGW); do
         rgw_name="client.rgw.${current_port}"
 
-        ceph_adm auth get-or-create $rgw_name \
-            mon 'allow rw' \
-            osd 'allow rwx' \
-            mgr 'allow rw' \
-            >> "$keyring_fn"
+        if [ "$rgw_dbstore" -eq 0 ] ; then
```

Comment:
In my multiple filters testing (https://github.com/ceph/ceph/pull/52665/files#diff-84b866e89a4af07ecc9905b4c7dc0be051d6327adf0dae8d4f817a22083fb330R1796) I have the check here as if number of mons != 0. Maybe that would make more sense over time once POSIX becomes a full backend.